alvr/sockets/src/control_socket.rs

```diff
@@ -62,7 +88,7 @@ fn framed_recv<R: DeserializeOwned>(
         }
 
         let packet_length =
-            FRAMED_PREFIX_LENGTH + u32::from_be_bytes(payload_length_bytes) as usize;
+            FRAMED_PREFIX_LENGTH + u32::from_le_bytes(payload_length_bytes) as usize;
```

Comment:
`resize()` also doesn't change the underlying allocation if the resize is smaller, so might as well just unconditionally `resize()` to the desired length and then have the vector itself be the source of truth for the size.

(ik this comment isn't exactly where the code is but github forces me to do it this way)

Reply:
Yes but it does matter too much i think. Just sending a slice at the end instead would be one less line of code